src/sdk/model/edge/edge.ts

```diff
@@ -384,15 +384,28 @@ export class Edge extends Entity {
   }
 
   /**
-   * Update the static routing service.
+   *  Update the static routing service.
    * @param {StaticRouting} request
    * @returns {Promise<Task>}
    */
   /* istanbul ignore next: autogenerated */
   async updateStaticRouting(request: StaticRouting): Promise<Task> {
     return Iland.getHttp().post(`/edges/${this.uuid}/actions/update-static-routing`, request.json).then((response) => {
-        const json = response.data as TaskJson;
-        return new Task(json);
+      const json = response.data as TaskJson;
+      return new Task(json);
+    });
+  }
+
+  /**
+   * Update the DHCP service.
+   * @param {Dhcp} request
+   * @returns {Promise<Task>}
+   */
+  /* istanbul ignore next: autogenerated */
+  async updateDHCP(request: Dhcp): Promise<Task> {
```

Comment:
this endpoint doesn't have the same model for GET and POST, you should have a DhcpServiceUpdateRequest model here